config/peerpods/podvm/lib.sh

```diff
@@ -457,6 +461,89 @@ function validate_podvm_image() {
     echo "Checksum of the PodVM image: $(sha256sum "$image_path")"
 }
 
+# Function to convert local bootc container image to cloud image
+# Input:
+# 1. container_image_repo_url: The registry URL of the source container image.
+# 2. image_tag: The tag of the source container image.
+# 3. auth_json_file (can be empty): Path to the registry secret file to use for downloading the image.
+# 4. run_args: arguments for the podman run command that runs bootc image builder
+# 5. bib_args: arguments for bootc image builder itself
+# Output: based on input, either, qcow2 image at output/qcow2/disk.qcow2 or ami
+# see: https://github.com/osbuild/bootc-image-builder
+function bootc_image_builder_conversion() {
+    container_image_repo_url="${1}"
+    image_tag="${2}"
+    auth_json_file="${3}"
+    run_args="${4}"
+    bib_args="${5}"
+
+    # some VM customizations
+    if [[ -n "${BOOTC_BUILD_CONFIG}" ]]; then
+        echo "Using Custom Bootc Build Configuration"
+        echo "${BOOTC_BUILD_CONFIG}" >> ./config.toml
+    else
+        echo "Using Default Bootc Build Configuration"
```

Comment:
Why don't you use the [config.toml](https://github.com/openshift/sandboxed-containers-operator/pull/484/commits/d6b6cceec908fc95e7509203f200bb42cfd89434#diff-a8633a2cbbc3a90771c8cbd93128222dd9007c8f97989b5fcd8429e8e4958e81) versioned on this repo?

I would also rename it to `bootc-config.toml` as `config.toml` is pretty generic and it might end up overwriting a file you don't want to (and introducing a silent bug).